ivshmem/Device.c

```diff
@@ -167,55 +161,71 @@ NTSTATUS IVSHMEMEvtDevicePrepareHardware(_In_ WDFDEVICE Device, _In_ WDFCMRESLIS
                 }
             }
             else
-            // shared memory resource
-            if ((deviceContext->interruptCount == 0 && memIndex == 1) || memIndex == 2)
-            {
-                deviceContext->shmemAddr.PhysicalAddress = descriptor->u.Memory.Start;
-                deviceContext->shmemAddr.NumberOfBytes = descriptor->u.Memory.Length;
-                DEBUG_INFO("memIndex = %d pa = %llx (%llx) size = %lx (%lx)", memIndex, descriptor->u.Memory.Start.QuadPart, deviceContext->shmemAddr.PhysicalAddress.QuadPart, descriptor->u.Memory.Length, deviceContext->shmemAddr.NumberOfBytes);
+                // shared memory resource
+                if ((deviceContext->interruptCount == 0 && memIndex == 1) || memIndex == 2)
+                {
+                    deviceContext->shmemAddr.PhysicalAddress = descriptor->u.Memory.Start;
+                    deviceContext->shmemAddr.NumberOfBytes = descriptor->u.Memory.Length;
+                    DEBUG_INFO("memIndex = %d pa = %llx (%llx) size = %lx (%lx)",
+                               memIndex,
+                               descriptor->u.Memory.Start.QuadPart,
+                               deviceContext->shmemAddr.PhysicalAddress.QuadPart,
+                               descriptor->u.Memory.Length,
+                               deviceContext->shmemAddr.NumberOfBytes);
 
 #if (NTDDI_VERSION >= NTDDI_WIN8)
-                result = MmAllocateMdlForIoSpace(&deviceContext->shmemAddr, 1, &deviceContext->shmemMDL);
+                    result = MmAllocateMdlForIoSpace(&deviceContext->shmemAddr, 1, &deviceContext->shmemMDL);
 #else
```

Comment:
@YanVugenfirer 

Looks like a part for refactoring. This condition can be removed as we drop Win8 support.

Reply:
@Jedoku ignore this comment